src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -66,6 +67,15 @@ export function PharmacyCard(props: PharmacyProps) {
       </Banner>
     );
   };
+
+  const isPopup = (organizationId: number) => organizationId === 25;
+
+  const popupWarningMarkup = isPopup(props.organizationId) ? (
+    <Banner title="This is a pop-up clinic." status="warning">
+      Please visit the website to confirm you are eligible for this location.
```

Comment:
Please translate with i18next - use t() function. We can add to the translation spreadsheet.